server.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+const cors = require('cors');
+const helmet = require('helmet');
+const session = require('express-session')
+const User = require('./auth/UserModel');
+
+// mongoose.connect('mongodb://localhost/cs10').then(() => {
+//   console.log('\n*** Connected to database ***\n');
+// });
+
+const server = express();
+
+server.use(cors());
+server.use(express.json());
+
+server.use('/api/user', userRouter);
+
+const session = require('express-session');
+
+server.use(
+  session({
+    secret: 'nobody toss a dwarf!',
+    cookie: {
+      maxAge: 1 * 24 * 60 * 60 * 1000
+    },
+    httpOnly: true,
+    secure: true,
```

Comment:
This isn't what was breaking your code, but remember that this needs to be false in development and change to true in production.